packages/pwa-kit-dev/package.json

```diff
@@ -25,7 +25,7 @@
   ],
   "scripts": {
     "build": "cross-env NODE_ENV=production internal-lib-build build",
-    "build:watch": "watch 'npm run build' src/ bin/",
+    "build:watch": "nodemon --watch 'src/**' --exec 'npm run build'",
```

Comment:
The original script watched the `bin` folder too. We should do that too with nodemon.

Reply:
Good catch! I've fixed that